src/service/performance-impl.js

```diff
@@ -665,6 +666,9 @@ export class Performance {
       data['value'] = this.timeOrigin_ + delta;
     }
 
+    // Emit events. Used by `gulp performance`.
+    self.dispatchEvent(createCustomEvent('csi', {detail: {label, delta}}));
```

Comment:
Nit: `'perf'` instead of `'csi'`? The latter is kind of overloaded.

Reply:
sure